app/controllers/holds_controller.rb

```diff
@@ -6,7 +6,6 @@ class HoldsController < ApplicationController
   before_action :check_for_blanks!, only: :create
   rescue_from NewHoldException, with: :deny_new
   rescue_from HoldCreateException, with: :deny_create
-  rescue_from HoldException, with: :past_date
```

Comment:
I think this is not used anymore 🤔  so I removed it

Reply:
Yeah I think it and it's model can be deleted, good eyes!